### PR TITLE
Update Docs SDK versions

### PR DESCRIPTION
**Pull request by acekyd**
This pull request primarily involves updates to various packages and dependencies across multiple files. The most significant changes include updates to the versions of several dependencies in `package.json` and `sdk-versions.json`, addition of a new dependency `web5-keymanager-aws` in various files, and changes to the key management tests.

Dependency Updates:

* [`package.json`](diffhunk://#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L37-R46): Updated the versions of several dependencies including `@tbdex/http-client`, `@tbdex/http-server`, `@tbdex/protocol`, `@web5/common`, `@web5/credentials`, `@web5/crypto`, `@web5/crypto-aws-kms`, `@web5/dids`, and `@web5/identity-agent`.
* [`sdk-versions.json`](diffhunk://#diff-ac32dca8fe30343cb10ab8b5ab1cb872ad6409e409814e983cbb0b3ef418e0dcL3-R18): Updated the versions of several dependencies in both the `js` and `jvm` sections.

Addition of new dependency:

* [`site/docs/web5/build/decentralized-identifiers/key-management.mdx`](diffhunk://#diff-83dafc02d54448c17f0398fd3c40c396dca039e0cf578a27d052091a2330a988L61-R62): Added `web5-keymanager-aws` as a dependency for `maven` and `gradle`.
* [`site/testsuites/testsuite-kotlin/pom.xml`](diffhunk://#diff-db6167e7c768e2ae7aae7099ca83c618c3fda9da4bdfab49ab36188e419baf93L18-R24): Added `web5-keymanager-aws` as a dependency and updated the version of `xyz.block.tbdex`. [[1]](diffhunk://#diff-db6167e7c768e2ae7aae7099ca83c618c3fda9da4bdfab49ab36188e419baf93L18-R24) [[2]](diffhunk://#diff-db6167e7c768e2ae7aae7099ca83c618c3fda9da4bdfab49ab36188e419baf93R100-R110)
* [`site/testsuites/testsuite-kotlin/src/test/kotlin/docs/web5/build/decentralizedidentifiers/KeyManagementTest.kt`](diffhunk://#diff-14a70da49fddacd30b16cfb34138b67c3d152e62bc616511d5150645e237448fL7-R7): Imported `web5.sdk.keymanager.aws.AwsKeyManager` instead of `web5.sdk.crypto.AwsKeyManager`.

Key Management Test Changes:

* [`site/testsuites/testsuite-javascript/__tests__/web5/build/decentralized-identifiers/key-management.test.js`](diffhunk://#diff-fafddb44a489f558961f42e95d029d6324683fc74c76b43cc4e81695ebc2212eL33-R34): Changed the test for 'initialize key management' to check for properties `_algorithmInstances` and `_keyStore` instead of an instance of `LocalKeyManager`.

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1207704987048331

### CODE REVIEW

**Review by blackgirlbytes — APPROVED** [review]
from what I see, this looks right!

**Comment by angiejones** [comment]
some conflicts to resolve

image

File: site/docs/web5/build/decentralized-identifiers/key-management.mdx [hunk 1]
```diff
@@ -58,8 +58,8 @@ Below is an example that uses `AwsKeyManager` for production environments, and `
 
 <Dependencies languageDependencies={[
   { language: 'javascript', dependencies: ['@web5/dids', '@web5/crypto', '@web5/crypto-aws-kms'] },
-  { language: 'maven', dependencies: ['web5-dids', 'web5-crypto'] },
-  { language: 'gradle', dependencies: ['web5-dids', 'web5-crypto'] },
+  { language: 'maven', dependencies: ['web5-dids', 'web5-crypto', 'web5-keymanager-aws'] },
```
Comment (angiejones) [inline site/docs/web5/build/decentralized-identifiers/key-management.mdx:61]: good catch

**Review by blackgirlbytes — CHANGES_REQUESTED** [review]
hmm something with color mode provider is making your tests fail

**Comment by acekyd** [comment]
> hmm something with color mode provider is making your deployment fail

Yeah I merged main in here and it seems something broke. Looking into it.

**Review by blackgirlbytes — APPROVED** [review]

**Pull request merged**